tests/unit_tests/databases/ssh_tunnel/dao_tests.py

```diff
@@ -35,7 +35,7 @@ def test_create_ssh_tunnel():
         "password": "bar",
     }
 
-    result = SSHTunnelDAO.create(properties)
+    result = SSHTunnelDAO.create(properties, commit=False)
```

Comment:
@hughhhh I set `commit=False` to mimic the other tests in the `ssh_tunnel/commands` folder which use `CreateSSHTunnelCommand` that has `commit=False` by default. If we enable the commit in this test or in the command ones, they will fail because there's no database with `id = 1`. If an actual commit is required for the tests, I suggest opening a follow-up PR and fixing not only this test but the ones in the `commands` folder.